New package: ECLGraphColor v0.1.3

JuliaRegistrator opened the pull request:
- Registering package: ECLGraphColor
- Repository: https://github.com/PetrKryslUCSD/ECLGraphColor.jl
- Created by: @PetrKryslUCSD
- Version: v0.1.3
- Commit: a16194d9851e842f942334f64baf98a8677f4f64
- Reviewed by: @PetrKryslUCSD
- Reference: https://github.com/PetrKryslUCSD/ECLGraphColor.jl/commit/a16194d9851e842f942334f64baf98a8677f4f64#commitcomment-141733406
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6a795ba2a3ebf7b88cc3eee0c2d38871ff95cd30d76b253adf31effefd697814e4d66756fd1176c9ee4045926ff64623eafc50632a759eee7ca1d642e0385f1d7f88743733452c887a840390d267a9413c91ac5b5a41a9acfcc4c6e38bbe1cee5e07a25cce8c4a26394bcfedc591972057ebf4fe99f30bdd17b232b45f864278c133a15017497bfcf8ea70840a454b76fcf860fd07575f342cfa2974030dc9e44eb9bac3926fe93618eeff3c34325e15e -->

goerz commented:
[noblock] Could you add a small usage example to the README?

goerz commented:
Thanks for updating the README, but you wouldn't have had to change the version to create a new PR at https://github.com/JuliaRegistries/General/pull/106271

In general, triggering new versions during the initial waiting period is not possible, so I'm going to have to close this PR in favor of https://github.com/JuliaRegistries/General/pull/106271 (which restarts the 3-day timer).

Alternatively, you can roll back the change of version from `v0.1.4` to `v0.1.3` and retrigger the registration for `v0.1.3`. That should then update *this* PR, and I would close https://github.com/JuliaRegistries/General/pull/106271

PetrKryslUCSD commented:
Ok, I will roll back the version.
Thanks!

On Mon, May 6, 2024, 9:12 AM Michael Goerz ***@***.***> wrote:

> Thanks for updating the README, but you wouldn't have had to change the
> version to create a new PR at #106271
> <https://urldefense.com/v3/__https://github.com/JuliaRegistries/General/pull/106271__;!!Mih3wA!EnPw3FTM3qgK6jxImtctVYHTzFizcFR-EE8iWZpBdmzpJqONibppGl5MutbD2U7lLTW7OU1oJKDssAJBeRULjhho$>
>
> In general, triggering new versions during the initial waiting period is
> not possible, so I'm going to have to close this PR in favor of #106271
> <https://urldefense.com/v3/__https://github.com/JuliaRegistries/General/pull/106271__;!!Mih3wA!EnPw3FTM3qgK6jxImtctVYHTzFizcFR-EE8iWZpBdmzpJqONibppGl5MutbD2U7lLTW7OU1oJKDssAJBeRULjhho$>
> (which restarts the 3-day timer).
>
> Alternatively, you can roll back the change of version from v0.1.4 to
> v0.1.3 and retrigger the registration for v0.1.3. That should then update
> *this* PR, and I would close #106271
> <https://urldefense.com/v3/__https://github.com/JuliaRegistries/General/pull/106271__;!!Mih3wA!EnPw3FTM3qgK6jxImtctVYHTzFizcFR-EE8iWZpBdmzpJqONibppGl5MutbD2U7lLTW7OU1oJKDssAJBeRULjhho$>
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.com/v3/__https://github.com/JuliaRegistries/General/pull/106228*issuecomment-2096414383__;Iw!!Mih3wA!EnPw3FTM3qgK6jxImtctVYHTzFizcFR-EE8iWZpBdmzpJqONibppGl5MutbD2U7lLTW7OU1oJKDssAJBeTobdqax$>,
> or unsubscribe
> <https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/ACLGGWA65BOEXLFQCAXATR3ZA6TXRAVCNFSM6AAAAABHIISDJCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAOJWGQYTIMZYGM__;!!Mih3wA!EnPw3FTM3qgK6jxImtctVYHTzFizcFR-EE8iWZpBdmzpJqONibppGl5MutbD2U7lLTW7OU1oJKDssAJBeaU_TCz4$>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

JuliaRegistrator committed:
New package: ECLGraphColor v0.1.3

UUID: 9c1363d0-405f-4779-b1bc-67e54cdcac85
Repo: https://github.com/PetrKryslUCSD/ECLGraphColor.jl.git
Tree: 2bfb2091f08f1f84517e9d8659b2e9d1084661ab

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

goerz commented:
[noblock] Sounds good!

(You have to edit https://github.com/JuliaRegistries/General/pull/106228#issuecomment-2096424937 to contain `[noblock]`)
